src/modules/schoolingOverview/docs/it_IT.md

```diff
@@ -0,0 +1,21 @@
+Soprattutto nelle grandi associazioni, l'elenco dei corsi disponibili diventa rapidamente molto lungo. Se si prende parte a molti corsi di formazione, anche allora, si perde rapidamente traccia di ciò che sta succedendo.
+
+Con il modulo Panoramica del corso`cerchiamo di risolvere questo problema.
```

Comment:
There is a single additional ` here. Is one missing or this one too much?

Reply:
It was missing ` now it's `Panoramica del Corso`